Makefile

```diff
@@ -110,44 +110,31 @@ coverage:
 
 .PHONY: format
 format: $(GOIMPORTS)
-	go list -f '{{.Dir}}' ./... | xargs $(GOIMPORTS) -w -local helm.sh/helm
+	GO111MODULE=on go list -f '{{.Dir}}' ./... | xargs $(GOIMPORTS) -w -local helm.sh/helm
 
 # ------------------------------------------------------------------------------
 #  dependencies
 
-.PHONY: bootstrap
-bootstrap: vendor
-
-$(DEP):
-	go get -u github.com/golang/dep/cmd/dep
+# If go get is run from inside the project directory it will add the dependencies
+# to the go.mod file. To avoid that we change to a directory without a go.mod file
+# when downloading the following dependencies
 
 $(GOX):
-	go get -u github.com/mitchellh/gox
+	(cd /; GO111MODULE=on GO111MODULE=on go get -u github.com/mitchellh/gox)
```

Comment:
how come we need to `cd` to the root here? Is this a thing required for CI?

Reply:
if you run `go get` from within the project directory, even to install an app like gox, it will add it to the `go.mod` file and deal with the dependency tree for it. So, run `build-cross` locally and have it install gox and you've modified your `go.mod` file with gox and an updated dependency tree.

Reply:
Ahh. I see what you mean. Thanks!